static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-main-variant.js

```diff
@@ -8,8 +8,8 @@ export const signInGateMainVariant = {
 	author: 'Mahesh Makani',
 	description:
 		'Show sign in gate to 100% of users on 3rd article view of simple article templates, with higher priority over banners and epic. Main/Variant Audience.',
-	audience: 0.9,
-	audienceOffset: 0.0,
+	audience: 0.8999,
+	audienceOffset: 0.0001,
```

Comment:
If this has a direct dependency on another test, you can make that explicit!

```suggestion
	audience: 0.9 - signInGateMandatoryLongTestRun.audience,
	audienceOffset: signInGateMandatoryLongTestRun.audience,
```